tests/end-to-end/ui/11-admin.js

```diff
@@ -704,5 +704,223 @@ describe('[Administration]', () => {
 				});
 			});
 		});
+
+		describe('[Accounts]', () => {
+			before(() => {
+				admin.settingsSearch.setValue('accounts');
+				admin.accountsLink.waitForVisible(5000);
+				admin.accountsLink.click();
+				admin.settingsSearch.setValue('');
+			});
+
+			describe('default user preferences', () => {
+				before(() => {
+					if (admin.accountsButtonCollapseDefaultUserPreferences.isVisible()) {
```

Comment:
I'm wondering why this `if` is necessary here. Why are all the groups already expanded (and consequently the `expand` class does not exists in the html) when the test reach the Accounts section?

Reply:
I have no idea 😄